prototype_source/pt2e_quant_ptq.rst

```diff
@@ -1,4 +1,4 @@
-(prototype) PyTorch 2.0 Export Post Training Static Quantization
+(prototype) PyTorch 2 Export Post Training Static Quantization
```

Comment:
@jerryzh168 do we need to remove the `Static` in the title?

Reply:
yes I think so, @andrewor14 could you update the PR to remove static in the doc as well